fiducialMarker/fiducialMarkerTest.py

```diff
@@ -7,6 +7,26 @@
 class TestFiducialMarker(unittest.TestCase):
 
     @staticmethod
+    def test_get_dictionary():
+        """
+        compare public attributes of dictionary to confirm equality

```

Comment:
Dictionary doesn't have an object equality comparison, rather than checking each attribute?
Either way this works, and it's good!